admin_commands/tremove.js

```diff
@@ -0,0 +1,28 @@
+// tremove.js
+// Clears topic at input index.
+module.exports.description = "Clears topic at the index given."
+
+const topic0p = require("../lib/topic.js");
+
+module.exports.run = (client, message, args) => {
+	const channel = message.channel;
+
+	// Get the index of topic we want removed
+	indexToClear = parseInt(args);
+	currentTopics = client.topics;
+	let returnMessage = "";
+
+	if (isNaN(indexToClear)) {
+		returnMessage = "Please enter a proper integer.";
+	} else if (indexToClear >= currentTopics.length) {
+		returnMessage = "Index given exceeds the length of current list of topics.";
+	} else {
+		currentTopics[indexToClear] = false;
```

Comment:
Should this be `currentTopics[indexToClear][1] = false`?

Reply:
ty! good catch